staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -2723,8 +2723,25 @@ func describeService(service *corev1.Service, endpoints *corev1.Endpoints, event
 		w.Write(LEVEL_0, "Type:\t%s\n", service.Spec.Type)
 		w.Write(LEVEL_0, "IP:\t%s\n", service.Spec.ClusterIP)
 
-		if service.Spec.IPFamily != nil {
-			w.Write(LEVEL_0, "IPFamily:\t%s\n", *(service.Spec.IPFamily))
+		if service.Spec.IPFamilyPolicy != nil {
+			w.Write(LEVEL_0, "IPFamilyPolicy:\t%s\n", *(service.Spec.IPFamilyPolicy))
+		}
+
+		if len(service.Spec.ClusterIPs) > 0 {
```

Comment:
is `describe` output consider API?  If so, OK.  If not, maybe we should drop the singular from this output?  I'll ask sig CLI

Reply:
we followed the same approach for PodIPs

Reply:
Sig CLI (slack) seems to think it's OK to print one-or-the-other-but-not-both if we want.  What do you think?